invadedhouse/commands/rename.py

```diff
@@ -0,0 +1,29 @@
+from commands import Command
+
+
+class Rename(Command):
+
+    def do(self, text: str):
+        player = self.get_player()
+        house = self.get_house()
+        current_room = player.get_location()
+
+        if text is None:
+            name = input('What\'s your name? > ')
```

Comment:
so, now I only do this if the user didn't already enter in their new name.

try entering something at the command prompt like `call me Daddy`